bin/deploy

```diff
@@ -132,12 +134,17 @@ am_I_sourced() {
   fi
 }
 
+
+
 main() {
   config_curl
 
   BLEED_DEPLOY=false
 
+  ENVREF=""
+
   parse_args "$@"
+  FIXED_ENVREF=${ENVREF//[.\/]/-}
```

Comment:
The reference provided may be a branch with slashes. I am using that reference to create a folder where to store the corresponding environment file. This is just to try to have the folder name cleaner in the filesystem. It may be necessary to add more substitutions.

Reply:
OK, but could you add a comment like `# Create safe directory name from git ref`?